src/constants/boxShadows/primary.ts

```diff
@@ -0,0 +1,9 @@
+const PRIMARY_BOX_SHADOWS = {
+  clickable: '0px 8px 24px rgba(52, 51, 82, 0.05)',
+  clickableHover: '0px 8px 24px rgba(52, 51, 82, 0.10)',
+  modalMobile: '0px -8px 24px rgba(52, 51, 82, 0.05)',
+  focus: '0px 0px 0px 2px #FFFFFF, 0px 0px 0px 4px #332E54',
```

Comment:
decided to use exact values here because I had them accessed via theme before but the `rgba` values above didn't have a theme color counter part and it look incosistent

Reply:
I think we should prefer to use the variables, no? The `rgba` values are fine since this is the only usage but the usage of `white` and `primary` in here is much clearer with the variables.

Reply:
I changed to variables, what is triggering me is that rgb(52, 51, 82) used above is #332E54  and looks unconsistent